From 853187d759eb3d5cd49eaba3e6d151ba05234df8 Mon Sep 17 00:00:00 2001
From: Fabrizio Castro <fabrizio.castro.jz@renesas.com>
Date: Thu, 7 Oct 2021 10:45:08 +0100
Subject: [PATCH] [HACK] firmware-pack: Add missing dependency

firmware-pack uses files generated by trusted-firmware-a,
which are deployed to DEPLOY_DIR_IMAGE, however,
firmware-pack.do_compile does not depend on
trusted-firmware-a.do_deploy.
A proper solution would be for firmware-pack to access those
files from its recipes-sysroot, but this quick fix will prevent
things from breaking during a build, while waiting for the
proper solution to become available.

Signed-off-by: Fabrizio Castro <fabrizio.castro.jz@renesas.com>
---
 recipes-bsp/firmware-pack/firmware-pack.bb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/recipes-bsp/firmware-pack/firmware-pack.bb b/recipes-bsp/firmware-pack/firmware-pack.bb
index 6a8848ac..ed3b4ddd 100644
--- a/recipes-bsp/firmware-pack/firmware-pack.bb
+++ b/recipes-bsp/firmware-pack/firmware-pack.bb
@@ -7,6 +7,7 @@ inherit deploy
 
 DEPENDS = "trusted-firmware-a u-boot"
 DEPENDS += " bootparameter-native fiptool-native"
+do_compile[depends] += "trusted-firmware-a:do_deploy"
 
 S = "${WORKDIR}"
 
-- 
2.25.1

